site/_resources/upgrading.md

```diff
@@ -335,4 +334,19 @@ and checking the annotations that store exact details using
 
 ```
 $ kubectl get configmap -n heptio-contour -o yaml contour
-```
\ No newline at end of file
+```
+
+[1]: {{site.github.repository_url}}/blob/v1.0.0/examples
+[2]: {% link docs/v1.0.0/deploy-options.md %} 
+[3]: {% link docs/v1.0.0/httpproxy.md %}
+[4]: {% link docs/v1.0.0/annotations.md %}
+[5]: {{site.github.repository_url}}/issues/899
+[6]: {% link docs/v1.0.0/configuration.md %}
+[7]: {{site.github.repository_url}}/blob/v1.0.0/examples/contour/README.md
+[8]: {{site.github.repository_url}}/blob/v1.0.0/examples/contour/02-rbac.yaml#L71
+[9]: {{site.github.repository_url}}/blob/v1.0.0/examples/contour/02-rbac.yaml
+[10]:https://www.envoyproxy.io/docs/envoy/v1.11.2/intro/version_history
+[11]:{{site.github.repository_url}}/blob/v0.15.3/examples/
+[12]:{% link docs/v1.0.0/deploy-options.md %}
+[13]:https://hub.docker.com/r/projectcontour/contour
+[14]:{% link docs/v1.0.0/grpc-tls-howto.md %}
```

Comment:
Add a space after the `:` for consistency?

Reply:
Will do